cmd/internal/config.go

```diff
@@ -40,10 +35,20 @@ func ConfiguredAndV1() bool {
 		log.Fatalf("failed to read the configuration file: %v", err)
 	}
 
-	return projectConfig.IsV1()
+	return true
 }
 
-func PrintV1DeprecationWarning() {
-	fmt.Printf(noticeColor, "[Deprecation Notice] The v1 projects are deprecated and will not be supported beyond "+
-		"Feb 1, 2020.\nSee how to upgrade your project to v2: https://book.kubebuilder.io/migration/guide.html\n")
+// IsVersionSupported returns true if the project is already configured and it is v2
+func IsVersionSupported() bool {
+	projectConfig, err := config.Read()
+
+	if os.IsNotExist(err) {
+		return false
+	}
+
+	if err != nil {
+		log.Fatalf("failed to read the configuration file: %v", err)
+	}
+
+	return projectConfig.IsV2()
```

Comment:
```suggestion
	return !projectConfig.IsV1()
```

This way we will be ready for v3